include/nbl/builtin/hlsl/sampling/uniform.hlsl

```diff
@@ -34,6 +35,12 @@ struct UniformHemisphere
     {
         return T(1.0) / (T(2.0) * numbers::pi<T>);
     }
+
+    template<typename U>
+    static sampling::quotient_and_pdf<U, T> quotient_and_pdf()
+    {
+        return sampling::quotient_and_pdf<U, T>::create(hlsl::promote<U>(1.0), pdf());
```

Comment:
you're already in the `sampling` namespace

Reply:
Yeah but compiler (or msvc at least) confuses the type name with the function name